redash/schedule.py

```diff
@@ -11,26 +15,36 @@
                           cleanup_query_results,
                           version_check, send_aggregated_errors)
 
+logger = logging.getLogger(__name__)
+
 rq_scheduler = Scheduler(connection=rq_redis_connection,
                          queue_name="periodic",
                          interval=5)
 
+def job_id(kwargs):
+    metadata = kwargs.copy()
+    if 'func' in metadata:
+        metadata['func'] = metadata['func'].__name__
+
+    return hashlib.sha1(json.dumps(metadata, sort_keys=True).encode()).hexdigest()
 
-def schedule(**kwargs):
+
+def prep(kwargs):
     interval = kwargs['interval']
     if isinstance(interval, timedelta):
-        interval = interval.seconds
+        interval = int(interval.total_seconds())
```

Comment:
What's the diff between `total_seconds()` and `seconds`?

Reply:
`timedelta(days=1).seconds == 0`

Reply:
:open_mouth: